faq.html

```diff
@@ -87,15 +87,15 @@ <h4 class="panel-title">
               <div class="panel-body">
                 <p>Open the drawer by swiping from the left or tapping the <i class="fa fa-bars"></i> burger icon on the top left corner. Go to <i class="fa fa-plus"></i> <b>Add Podcast</b>.<br />Now, you can either 
                   search the iTunes catalog, browser gpodder.net or add a podcast by URL.</p>
-                  <p>You can also click RSS links (and other protocols) from the browser and select AntennaPod to open them.</p>
+                  <p>You can also import feeds by opening protocols such as RSS with AntennaPod.</p>
```

Comment:
That being nitpicky is in my character doesn't mean that you need to copy this behaviour ;)

Concerning 'the right terms': on [this lovely website](http://www.nu.nl/internet/3946198/leukste-podcasts-van-internet-en-zo-abonneer-je.html) I long-press the link to [this RSS feed](http://feeds.feedburner.com/movieinsiders) and chose to 'open with AntennaPod' (or share -> add podcast). From a user perspective I don't download/interact with a file (at least don't experience it as such). Should we still technically correct terms like 'files and intent filters'?

But you're right, on a technical level. Suggestions very much welcome, I've been struggling with this one :)